ktor-server/ktor-server-host-common/jvm/test/io/ktor/tests/hosts/CommandLineTest.kt

```diff
@@ -88,6 +88,25 @@ class CommandLineTest {
         assertEquals("8081", port)
     }
 
+    @Test
+    fun testListPropertiesConfig() {
```

Comment:
Please add test for config lists such as 
```
users:
    - name: test
      password: asd
    - name: other
      password: qwe
```
or in string representation
```
users.0.name=test
users.0.password=asd
users.1.name=other
users.1.password=qwe
```

Reply:
Didn't know about this case, thanks for spotting